fix: pin upstream firmware to git commit

twz123 opened the pull request:
The previous tag has been deleted? The files didn't change, though: the hashes still match.

ryan4yin commented:
Indeed, it's possible for tag to be deleted. would you please change git tag to commit id? In this way, the problem can be completely avoided.

twz123 committed:
fix: pin upstream firmware to git commit

The previous tag has been deleted? The files didn't change, though: the
hashes still match.
